### PR TITLE
test: Login/logout tests

### PR DESCRIPTION
**Pull request by bsilkyn**
These are some e2e tests regarding the login and logout routine

### CODE REVIEW

File: backend/authentication/views.py [hunk 1]
```diff
@@ -59,3 +62,50 @@ def echo(self, request: Request) -> Response:
...
         raise AuthenticationFailed(token_serializer.errors)
+
+
+data = {
+    "id": "1234",
+    "username": "test",
+    "email": "test@test",
+    "first_name": "test",
+    "last_name": "test",
+}
```
Comment (Topvennie) [inline backend/authentication/views.py:73]: I'd prefer to see this in `ypovoli.settings.py`.
Keep all the configuration together.

File: frontend/src/test/e2e/login.cy.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        cy.visit('/');
+        // should redirect to login page
+        cy.url().should('match', /^.*\/auth\/login$/);
+    });
+    // Next test is commented out because it loads endlessly when clicking login button
+    // it('login button redirects to correct external website', () => {
+    //     cy.visit('/');
+    //     cy.get('#login').click();
+    //     cy.url().should('match', /^$/)
+    // })
```
Comment (Topvennie) [inline frontend/src/test/e2e/login.cy.ts:13]: Can this be removed or is it something you want to look at a later date?

**Review by Topvennie — CHANGES_REQUESTED** [review]

File: frontend/src/router/guards/authentication.guard.ts [hunk 3]
```diff
@@ -10,6 +10,7 @@ export async function AuthenticationGuard(to: RouteLocationNormalized): Promise<
     const { refreshUser } = useAuthStore();
     const { intent, isAuthenticated } = storeToRefs(useAuthStore());
 
+    console.log(to.name as string);
```
Comment (DeLany123) [inline frontend/src/router/guards/authentication.guard.ts:13]: Can you delete this please.

Comment (bsilkyn) [reply frontend/src/router/guards/authentication.guard.ts:13]: well spotted

**Pull request merged**